doc/ui-extensions-sdk-frontend.md

```diff
@@ -1,31 +1,31 @@
-# Widget API Reference
+# UI Extensions API Reference
 
-This document describes the API that a custom widget can use to
+This document describes the API that a custom extension can use to
 communicate with the Contentful Management App.
 
 ### Table of Contents
 * [Inclusion into your project](#inclusion-into-your-project)
 * [Initialization](#initialization)
-* [`widget.contentType`](#widgetcontenttype)
-* [`widget.field`](#widgetfield)
-* [`widget.entry`](#widgetentry)
+* [`extension.contentType`](#extensioncontenttype)
+* [`extension.field`](#extensionfield)
+* [`extension.entry`](#extensionentry)
   * [`entry.fields[name]`](#entryfieldsname-field)
```

Comment:
we say `entry.fields[name]` here but in the section for this we say `entry.fields[id]`. We should choose one. Also, this hyperlink doesn't work cuz of the name mismatch.